packages/arb-ts/src/lib/utils/networks.ts

```diff
@@ -177,33 +179,109 @@ export const l2Networks: L2Networks = {
   },
 }
 
-const addCustomNetwork = ({
-  customL1Network,
-  customL2Network,
-}: {
+export interface CustomNetworks {
   customL1Network?: L1Network
```

Comment:
is there a case in which a consumer wouldn't provide both custom networks?
```suggestion
  customL1Network: L1Network
  customL2Network: L2Network
```